wlutil/initramfsRoot/init

```diff
@@ -0,0 +1,13 @@
+#!/bin/busybox sh
+
+/bin/busybox sh setupSystem.sh
+if [ $? != 0 ]; then
+  echo "Failed to setup system. Dropping to emergency shell."
+  exec /bin/busybox sh
```

Comment:
With luck, this may make things a bit more robust for us when we boot things and the block device or early boot breaks or something.